flink/v1.16/flink/src/test/java/org/apache/iceberg/flink/source/TestFlinkTableSource.java

```diff
@@ -97,6 +97,7 @@ public void before() {
   public void clean() {
     sql("DROP TABLE IF EXISTS %s.%s", DATABASE_NAME, TABLE_NAME);
     sql("DROP DATABASE IF EXISTS %s", DATABASE_NAME);
+    dropCatalog(CATALOG_NAME, true);
     sql("DROP CATALOG IF EXISTS %s", CATALOG_NAME);
```

Comment:
I think this line needs to be removed, given that we call dropCatalog above

Reply:
Thanks for catching this